index.html

```diff
@@ -1,11 +1,43 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <meta charset="UTF-8">
     <link rel="stylesheet" href="styles.css">
     <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@400;700&display=swap" rel="stylesheet">
   </head>
   <body>
-  
+    <h1 class="heading">✨ Self Care Center ✨</h1>
+    <br>
```

Comment:
Consider using CSS to create space between your HTML elements instead of the HTML language's built in line breaks